it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/bigtable/BigtableResourceManager.java

```diff
@@ -262,7 +270,10 @@ private void checkHasTable(String tableId) {
    */
   public synchronized void createTable(String tableId, Iterable<String> columnFamilies)
       throws BigtableResourceManagerException {
-    createTable(tableId, columnFamilies, Duration.ofHours(1));
+    BigtableTableSpec spec = new BigtableTableSpec();
+    spec.setColumnFamilies(columnFamilies);
+    spec.setMaxAge(Duration.ofHours(1));
```

Comment:
No, 1h is because it was 1h before. 

You can use another constructor 

``public synchronized void createTable(String tableId, BigtableTableSpec bigtableTableSpec)``

And pass ``maxAge`` via ``bigtableTableSpec``, it allows to set any ``maxAge``.